client/src/components/HistoryViewer/HistoryViewerVersionDetail.js

```diff
@@ -8,22 +8,74 @@ import { versionType } from 'types/versionType';
 
 class HistoryViewerVersionDetail extends PureComponent {
   componentWillMount() {
-    this.toggleToolbarClass();
+    this.toggleToolbarClass(true);
   }
 
   componentWillUnmount() {
-    this.toggleToolbarClass();
+    this.toggleToolbarClass(false);
+  }
+
+  /**
+   * A useful doc block explaining why we construct an array of versions in each case (noting that
```

Comment:
I didn't expect you to copy and paste this =)

Reply:
😂 In my defence, I did _intend_ to edit it afterwards! 😅